log4j-api/src/main/java/org/apache/logging/log4j/message/MapMessage.java

```diff
@@ -200,7 +200,7 @@ public void put(final String key, final String value) {
      * @param map The Map to add.
      */
     public void putAll(final Map<String, String> map) {
```

Comment:
perhaps this should take `Map<String,  V>` instead. I'd prefer to make that change separately though